project/CommonSettings.scala

```diff
@@ -267,4 +268,110 @@ object CommonSettings {
 
   lazy val binariesPath: Path =
     Paths.get(Properties.userHome, ".bitcoin-s", "binaries")
+
+
+  //see https://www.scala-sbt.org/sbt-native-packager/archetypes/jlink_plugin.html?highlight=jlinkignore#jlink-plugin
+  val jlinkIgnore = {
+    val deps = Vector(
+      //direct_dependency -> transitive dependency
```

Comment:
you should look through these, what `jlinkIgnore` does it tell `jlink` its ok that these transitive dependencies are missing at runtime. This _assumes_ that none of the code in these transitive dependencies gets executed during runtime.

I _believe_ if code is attempted to be executed at runtime, and it is missing (if you put it in `jlinkIgnore`) the jvm will bomb. So look through these and include the ones you need as explicit dependencies in `build.sbt`